tests/fixtures/mariadb_fixtures.py

```diff
@@ -0,0 +1,145 @@
+import logging
+from collections import Generator
+from typing import List, Dict
+from uuid import uuid4
+
+import pytest
+from sqlalchemy.orm import Session
+
+from fidesops.db.session import get_db_session, get_db_engine
+from fidesops.models.connectionconfig import ConnectionConfig, ConnectionType, AccessLevel
+from fidesops.models.datasetconfig import DatasetConfig
+from fidesops.service.connectors import MariaDBConnector
+from fixtures.application_fixtures import integration_secrets
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def connection_config_mariadb(db: Session) -> Generator:
+    connection_config = ConnectionConfig.create(
+        db=db,
+        data={
+            "name": str(uuid4()),
+            "key": "my_maria_db_1",
+            "connection_type": ConnectionType.mariadb,
+            "access": AccessLevel.write,
+            "secrets": integration_secrets["mariadb_example"],
+        },
+    )
+    yield connection_config
+    connection_config.delete(db)
+
+
+@pytest.fixture
+def mariadb_example_test_dataset_config(
+        connection_config_mariadb: ConnectionConfig,
+        db: Session,
+        example_datasets: List[Dict],
+) -> Generator:
+    mariadb_dataset = example_datasets[6]
+    fides_key = mariadb_dataset["fides_key"]
+    connection_config_mariadb.name = fides_key
+    connection_config_mariadb.key = fides_key
+    connection_config_mariadb.save(db=db)
+    dataset = DatasetConfig.create(
+        db=db,
+        data={
+            "connection_config_id": connection_config_mariadb.id,
+            "fides_key": fides_key,
+            "dataset": mariadb_dataset,
+        },
+    )
+    yield dataset
+    dataset.delete(db=db)
+
+
+@pytest.fixture(scope="function")
+def mariadb_integration_session(connection_config_mariadb):
+    example_mariadb_uri = MariaDBConnector(connection_config_mariadb).build_uri()
+    engine = get_db_engine(database_uri=example_mariadb_uri)
+    SessionLocal = get_db_session(engine=engine)
+    yield SessionLocal()
+
+@pytest.fixture(scope="function")
+def mariadb_integration_db(mariadb_integration_session):
```

Comment:
currently unused, but will be in future once Mariadb query generation is implemented

Reply:
One step better from here (and what I'm aiming for in the other PR) is to split these into fixtures functions themselves, such that you could have `mariadb_customer` as a fixture, and that generates a brand new customer row in the DB (and any such dependencies) off Faker generated data — that's a stretch goal here

Reply:
ok, mind if we follow-up with this one? I have yet to implement the mariadb query generation piece, in which I will properly test these fixtures